PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -1248,3 +1248,8 @@ enum Platform { PF_UNKNOWN, PF_X86, PF_X64, PF_IA64, PF_ARM64 };
 	//scnNotification->nmhdr.code = NPPN_NATIVELANGCHANGED;
 	//scnNotification->nmhdr.hwndFrom = hwndNpp
 	//scnNotification->nmhdr.idFrom = 0; // preserved for the future use, must be zero
+
+	#define NPPN_TOOLBARICONSETCHANGED (NPPN_FIRST + 32) // To notify plugins that toolbar icon set selection has changed
+	//scnNotification->nmhdr.code = NPPN_TOOLBARICONSETCHANGED;
+	//scnNotification->nmhdr.hwndFrom = hwndNpp;
+	//scnNotification->nmhdr.idFrom = 0;

```

Comment:
It's shame that plugins have to call NPPM_GETTOOLBARICONSETCHOICE after receiving this notification.
I would suggest to use idFrom to contain the icon set choice:
```cpp
//scnNotification->nmhdr.idFrom = iconSetChoice;
	// where iconSetChoice could be 1 of 5 possible values:
	// 0 (Fluent UI: small), 1 (Fluent UI: large), 2 (Filled Fluent UI: small), 3 (Filled Fluent UI: large) and 4 (Standard icons: small).
```

Reply:
Totally agree. I have done so in the 2nd commit.